src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/CalendarController.java

```diff
@@ -0,0 +1,57 @@
+package com.greenfoxacademy.opal.kalendaryo.kalendaryo.controllers;
+
+import com.google.api.services.calendar.model.Calendar;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+
+import static com.greenfoxacademy.opal.kalendaryo.kalendaryo.authorization.AuthorizeKal.getCalendarService;
+
+@Controller("/calendar")
+public class CalendarController {
+
+    @GetMapping("/show")
+    public String showCalendar() throws IOException {
+        com.google.api.services.calendar.Calendar service =
+                getCalendarService();
+
+        com.google.api.services.calendar.model.Calendar calendar =
```

Comment:
We still have this import information on the variable, let's remove it everywhere and put among imports